test/e2e/network/service.go

```diff
@@ -3779,12 +3783,29 @@ func validateEndpointsPorts(c clientset.Interface, namespace, serviceName string
 			// Retry the error
 			return false, nil
 		}
-		portsByPodUID := e2eendpoints.GetContainerPortsByPodUID(ep)
+		endpointsPortsByPodUID := portsByPodUID(e2eendpoints.GetContainerPortsByPodUID(ep))
+
+		opts := metav1.ListOptions{
+			LabelSelector: "kubernetes.io/service-name=" + serviceName,
+		}
+		es, err := c.DiscoveryV1beta1().EndpointSlices(namespace).List(context.TODO(), opts)
+		if err != nil {
+			framework.Logf("Failed go list EndpointSlice objects: %v", err)
```

Comment:
Hmm looks like this is causing an issue in `pull-kubernetes-conformance-kind-ga-only-parallel`: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/93597/pull-kubernetes-conformance-kind-ga-only-parallel/1289156168177946628#1:build-log.txt%3A12459

Used here: https://github.com/kubernetes/kubernetes/blob/ee72e029070128447c673b75dbaedba978e31f48/test/e2e/network/service.go#L823

Reply:
This is interesting - the test claims that "EndpointSlice" type is not registered there, which doesn't make much sense...